fe/fe-core/src/main/java/com/starrocks/sql/optimizer/ChildPropertyDeriver.java

```diff
@@ -837,6 +837,7 @@ public Void visitPhysicalHiveScan(PhysicalHiveScanOperator node, ExpressionConte
     @Override
     public Void visitPhysicalSchemaScan(PhysicalSchemaScanOperator node, ExpressionContext context) {
         if (getRequiredLocalDesc().isPresent()) {
+            outputInputProps.add(new Pair<>(distributeRequirements(), Lists.newArrayList()));
```

Comment:
What’s this meaning?

Reply:
SchemaScanNode can satisfy any local property requirements